### PR TITLE
Add guest button

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
# What
Adds a "Guest" button to the welcome page

# Why
Good way for accessing the featured page without being signed in and might entice new users to checkout the instance more.

# Additional info (optional)
image

### CODE REVIEW

File: packages/client/src/pages/welcome.entrance.a.vue
Comment (Johann150) [inline packages/client/src/pages/welcome.entrance.a.vue]: If it is a link you can use `link` and `to` as seen in:
https://github.com/misskey-dev/misskey/blob/01d5a97a4fab5f311f977b897e779a1393c1cd05/packages/client/src/pages/my-antennas/index.vue#L5

```suggestion
					<MkButton :link="true" to="/featured" inline rounded style="margin-left: 12px;">{{ i18n.ts.guest }}</MkButton>
```

**Review by Johann150 — CHANGES_REQUESTED** [review]
I'm not sure why you named the button "guest" (which introduces a new internationalization string) instead of calling it just "explore" since that is where it goes to.

**Comment by ThatOneCalculator** [comment]
> I'm not sure why you named the button "guest" (which introduces a new internationalization string) instead of calling it just "explore" since that is where it goes to.

Honestly you're right, that's just what I was calling it before on my instance's patchset.

**Comment by robflop** [comment]
The purpose of this looks related to #8927.

**Review by Johann150 — CHANGES_REQUESTED** [review]
The `/featured` page has been removed in commit afe0d9a2663b541237a326b8dfbbb3d5070fa11f. I'm not sure what the appropriate replacement page is.

**Comment by ThatOneCalculator** [comment]
It's `/explore` 

image

**Review by Johann150 — APPROVED** [review]

**Comment by ThatOneCalculator** [comment]
Gonna draft this until #8927 is either merged or closed. If it's merged, I'll change `/explore` to `/timeline`.

**Comment by futchitwo** [comment]
> Gonna draft this until #8927 is either merged or closed. If it's merged, I'll change `/explore` to `/timeline`.

Note that if both local and global timelines are disabled, nothing will appear on the timeline page!

**Comment by ThatOneCalculator** [comment]
Ah, then maybe if both are disabled, automatically redirect it to `/explore`? Also, I'm not sure of any instances that have both disabled

**Comment by tamaina** [comment]
I don't think I should close this PR.

**Comment by aprilthepink** [comment]
well, it was implemented in calckey in the meantime, not sure if this is still up to date code?

**Comment by saschanaz** [comment]
Should I open a new PR for this? Not sure why this is not being merged

**Comment by aprilthepink** [comment]
> Should I open a new PR for this? Not sure why this is not being merged

thats a... dumb thing to propose? wouldnt that be just copying the authors work and you getting credit for it?

**Comment by saschanaz** [comment]
That was solely because this PR is currently stale. Dumb indeed if the author intends to maintain this!

**Comment by ThatOneCalculator** [comment]
I don't intend to maintain this PR, and it's only one line. Do with it what you will

**Comment by acid-chicken** [comment]
> > Should I open a new PR for this? Not sure why this is not being merged
>
> thats a... dumb thing to propose? wouldnt that be just copying the authors work and you getting credit for it?

I recommend learning the difference between Author and Committer in Git.

**Comment by saschanaz** [comment]
(Oops, sorry for the noise)

There's also a co-author thing in Git too.

**Comment by tamaina** [comment]
Close due to the unlikely atmosphere of merging.

**Pull request closed without merging**
